Tests/SentryTests/Transaction/SentryTransactionTests.swift

```diff
@@ -223,7 +228,9 @@ class SentryTransactionTests: XCTestCase {
     
 #if os(iOS) || os(macOS) || targetEnvironment(macCatalyst)
     func testTransactionWithContinuousProfile() throws {
-        SentrySDK.setStart(Options())
+        let options = Options()
+        options.profilesSampleRate = nil // enables continuous profiling
```

Comment:
Do you actually need to set this explicitly? I thought the default value was nil?

Reply:
yep missed this one